front/app/modules/commercial/content_builder/admin/components/contentBuilderToggle.tsx

```diff
@@ -0,0 +1,85 @@
+import QuillMultilocWithLocaleSwitcher from 'components/UI/QuillEditor/QuillMultilocWithLocaleSwitcher';
```

Comment:
The files for components should start with a capital letter - the file here should be called `ContentBuilderToggle.tsx`